lib/drive_downloader.rb

```diff
@@ -29,21 +29,13 @@ def json_list
       {"files" => list_files}.to_json
     end
 
-    def pick_file(file_id)
-      @file_id = "0B7WjYjWZJv_4blA0a2p6RzVraFE"
-      # click on a file from JsonFile sends a POST :file_id to create
-      # something like a <%= select_tag(:file_id) %>
-      # pick by file_id from the view
-      # google_files.file_id(picked)
-      # returns file_id
-    end
-
-    def create_from_id(file_id)
+    def download
```

Comment:
we need to talk about the name of this method. It sure does download the file, but its return value is the path... so it seems that the main behavior of this method is something like `download_path` or something